raiden/blockchain/events.py

```diff
@@ -144,7 +147,7 @@ def get_all_netting_channel_events(
         pyethapp_chain,
         netting_channel_address,
         events=ALL_EVENTS,
-        from_block=0,
+        from_block='earliest',
```

Comment:
I don't understand what you are trying to do with this PR. If you set `earliest` or `pending` (it's the same thing) as the default value, it is going to be from the last mined block. Source [here](https://github.com/ethereum/wiki/wiki/JSON-RPC#parameters-38). 

That way we would never be able to get past events. This is now how we want the events to work at the moment.